java-util/src/main/java/io/druid/java/util/common/granularity/NoneGranularity.java

```diff
@@ -42,23 +42,19 @@ public DateTimeFormatter getFormatter(Formatter type)
   @Override
   public DateTime increment(DateTime time)
   {
-    return new DateTime(time.getMillis() + 1);
+    return time.plus(1);
   }
 
   @Override
   public DateTime decrement(DateTime time)
   {
-    return new DateTime(time.getMillis() - 1);
+    return time.minus(1);
   }
 
   @Override
   public DateTime bucketStart(DateTime time)
   {
-    if (time == null) {
-      return null;
-    }
-
-    return new DateTime(time.getMillis());
+    return time;
```

Comment:
There is no guarantee that the chronology of `time` is equal to the return contract, right? should this method do any sort of "utc" enforcement?